docs/workforce-management-framework/topics/con-datasync-module.adoc

```diff
@@ -0,0 +1,42 @@
+= {WFM-RC-NameLong} Datasync Module
```

Comment:
I think this file was not to be added? After a brief look, it looks identical to the text in the adoc above.

Reply:
the rename left the original file. removed the original